### DISCUSSION TITLE
How to customize the NuxtWelCome component in Nuxt3 version?

### DISCUSSION THREAD

**Discussion by binbin0916** [nuxt/nuxt#30030] [Questions]
I want to customize the NuxtWelCome component to achieve a unified website style, but I couldn't find a way to modify this component on the official website. Has anyone made any modifications to this component?

**Comment by ndragun92** [comment]
![image](https://github.com/user-attachments/assets/6ab1073d-e197-4b57-8150-926dc836834e)

There are props but you cannot modify how template looks like since its designed to be a placeholder and not something that you should use.

If you want to get HTML of it you can copy and paste it from
node_modules/nuxt/dist/app/components/welcome.vue

Everything inside that template is hardcoded even if you try to modify props itself

Hope it helps! @binbin0916
